### PR TITLE
[Security] Update access_control.rst

### PR DESCRIPTION
**Pull request by BooleanType**
IMHO, rule 1 should be applied instead of rule 2, because URI and IP match is enough for rule 1.

### CODE REVIEW

**Comment by xabbuh** [comment]
I think a better solution would be to reorder the rules instead.

**Comment by BooleanType** [comment]
> I think a better solution would be to reorder the rules instead.

Thought about it too :) Done.

File: security/access_control.rst [hunk 1]
```diff
@@ -145,13 +145,13 @@ if ``ip``, ``port``, ``host`` or ``method`` are not specified for an entry, that
...
-| ``/admin/user`` | 127.0.0.1   | 80          | symfony.com | GET        | rule #1 (``ROLE_USER_IP``)     | The ``path`` and ``ip`` still match. This would also match  |
+| ``/admin/user`` | 127.0.0.1   | 80          | symfony.com | GET        | rule #2 (``ROLE_USER_IP``)     | The ``path`` and ``ip`` still match. This would also match  |
 |                 |             |             |             |            |                                | the ``ROLE_USER_HOST`` entry, but *only* the **first**      |
 |                 |             |             |             |            |                                | ``access_control`` match is used.                           |
 +-----------------+-------------+-------------+-------------+------------+--------------------------------+-------------------------------------------------------------+
-| ``/admin/user`` | 127.0.0.1   | 8080        | symfony.com | GET        | rule #2 (``ROLE_USER_PORT``)   | The ``path``, ``ip`` and ``port`` match.                    |
+| ``/admin/user`` | 127.0.0.1   | 8080        | symfony.com | GET        | rule #1 (``ROLE_USER_PORT``)   | The path, ip and port match.                                |
```
Comment (xabbuh) [inline security/access_control.rst]: Can you revert here so that the double backticks are back again?

**Review by xabbuh — APPROVED** [review]
Looks great apart from my little comment!

File: security/access_control.rst (hunk 1 above)
Comment (BooleanType) [reply security/access_control.rst]: Done :)

**Review by OskarStark — APPROVED** [review]

**Comment by OskarStark** [comment]
Thank you @BooleanType.

**Pull request merged**